content/en/agent/iot/_index.md

```diff
@@ -0,0 +1,159 @@
+---
+title: IoT Agent
+kind: documentation
+beta: true
+further_reading:
+  - link: "/getting_started/agent/"
+    tag: "Documentation"
+    text: "Getting Started with the Agent"
+---
+
+## Overview
+
+The Datadog IoT Agent is a version of the Agent optimized for monitoring IoT devices and embedded applications. Customers use the IoT Agent to monitor a wide variety of devices from digital displays to security devices running image detection algorithms.
+
+<div class="alert alert-warning">
+The IoT Agent is currently available for private beta. Request access by completing the <a href="https://www.datadoghq.com/">form</a>.
+</div>
+
+## Capabilities
+
+The IoT Agent includes the following system checks. Configuration is identical to the full Agent.
```

Comment:
I'm not a big fan of using "full Agent" in public-facing docs. The official name of the product is "Datadog Agent", but this might not be enough to distinguish it from the "Datadog IoT Agent"... I don't know how to solve this problem, though. I can only think of other adjectives like "regular Datadog Agent" or "main Datadog Agent", which I'm not sure are better.

Checking what others do, Facebook has a "Facebook Lite" product and they don't call the full app "full Facebook": https://www.facebook.com/help/1278187755533783

Reply:
I agree and also not sure how to handle this. @jimmycaputo

Reply:
It doesn't bother me that much but maybe a better way to make this point is: Configuration for IoT devices is identical to other types of hosts.

Reply:
Indeed, rewording can solve the problem, but note there are 3 occurrences of "full Agent" in this document, so we will need to reword each of them differently.

Reply:
I re-worded this one as Jimmy suggested and changed "full Agent" to "standard Agent" in the other places.